build.js

```diff
@@ -93,13 +93,35 @@ const buildActivities = async (activitiesFilter = [], activitiesGroup) => {
 	for (const activity of activitiesToBuild) {
 		let activityConfig = {};
 		const activityConfigPath = `./src/activities/${activity.activity}/activity_config.json`;
+		const packageJsonPath = `./src/activities/${activity.activity}/package.json`;
+		let packageJson;
 		try {
 			activityConfig = JSON.parse(fs.readFileSync(activityConfigPath, 'utf8'));
 		} catch (err) {}
 		const config = Object.keys(activityConfig).reduce((accum, curr) => {
 			accum[`process.env.${curr.toUpperCase()}`] = JSON.stringify(activityConfig[curr]);
 			return accum;
 		}, {});
+		// check if activity has package.json
+		if (fs.existsSync(packageJsonPath)) {
+			console.log('package json found');
+			const packageJson = JSON.parse(fs.readFileSync(packageJsonPath, 'utf-8'));
+			if (packageJson.scripts && packageJson.scripts.dist) {
+				console.log(`"dist" script found. Running "dist" script...`);
+				const dirPath = `./src/activities/${activity.activity}`;
+				const { exec } = await import('child_process')
+				exec('npm run dist', { cwd: dirPath }, (error, stdout, stderr) => {
+					if (error) {
+						console.error(`Error executing "dist" script: ${error.message}`);
+						return;
+					}
+					if (stderr) {
+						console.error(`stderr: ${stderr}`);
+					}
```

Comment:
I see console.log and console.error. It makes me wonder if logs could be sent somewere else if that is an error, something that alert us in Slack to say "deploy of {activityName} failed to do {error.message}". 
If I was to ask for a favor at this stage, it would be to include some activity identifier in each console.[log|error] message. The {activity.activity} might be enough

Reply:
@connortobrien To Jason's point, could we use `loggingError` here?

Reply:
@sambbaron The console statements would only be logged to the github action workflow for debugging.

Reply:
@wilsonweb let me see if these can get elevated to slack via github actions